pool/web/view/component/component_list.ml

```diff
@@ -119,17 +119,55 @@ let pagination language query { Pagination.page; page_count; _ } =
     ]
 ;;
 
-let search language query searchable_by =
-  [ Component_input.input_element
-      ?value:(query.search |> CCOption.map Search.query_string)
-      ~hints:
-        [ Pool_common.I18n.SearchByFields
-            (CCList.map Column.field searchable_by)
+let searchbar language query searchable_by =
```

Comment:
**suggestion**. Since this already has a `query`, you could get rid of the form entirely and do the same thing the `Sortable_table` does to send an hx-get request:

```ocaml
input ~a:[ a_name search_label 
         ; a_input_type `Search
         ; a_value (Option.map_or ~default:"" Search.query_string query.search)
         ]
         @ (hx_get ~url ~target)
         @ (hx_trigger "input changed delay:300ms, search")
```

Where `hx_get` is something like:

```ocaml
let hx_get ~url ~target =
  [ a_user_data "hx-get" url
  ; a_user_data "hx-push-url" url
  ; a_user_data "hx-target" target
  ; a_user_data "hx-swap" "outerHTML"
  ]
;;
```

and `hx_trigger` is very similarly setting the `hx-trigger` attribute.

Reply:
@leostera perfect thanks, I'll add this to the "integrate search in sortable table" issue.

Reply:
I've changed it already ;-)

Reply:
Excellent!